docs/source/docker/getting_started/selection.rst

```diff
@@ -0,0 +1,518 @@
+.. _worker-selection:
+
+Selection
+=========
+
+Lightly allows you to specify the subset to be selected based on several objectives.
+
+E.g. you can specify that the images in the subset should be visually diverse, be images the model struggles with (Active Learning),
+should only be sharp images, or have a certain distribution of classes, e.g. be 50% from sunny, 30% from cloudy and 20% from rainy weather.
+
+Each of these objectives is defined by a `strategy`. A strategy consists of two parts:
+
+- The :code:`input` defines which data the objective is defined on. This data is either a scalar number or a vector for each sample in the dataset.
+- The :code:`strategy` itself defines the objective to apply on the input data.
+
+Lightly allows you to specify several objectives at the same time. The algorithms try to fulfil all objectives simultaneously.
+
+Lightly's data selection algorithms are supporting four types of input:
+
+- **Embeddings** computed using `our open source framework for self-supervised learning <https://github.com/lightly-ai/lightly>`_
```

Comment:
It's visually unappealing that some of the points are bold and others are links

Reply:
Igor drafted this and I think that it is OK.

Reply:
Alternatively, we could also delete it, as it contains the same information as later in the detailed input description.
Perhaps @IgorSusmelj has an opinion here.

Reply:
your call 🙂

Reply:
I would keep it like this and change it when we see the need for it.